neo/Network/P2P/Connection.cs

```diff
@@ -14,7 +14,7 @@ internal class Ack : Tcp.Event { public static Ack Instance = new Ack(); }
 
         public IPEndPoint Remote { get; }
         public IPEndPoint Local { get; }
-        public abstract int ListenerPort { get; }
+        public abstract int ListenerTcpPort { get; }
```

Comment:
This is for tcp only?

Reply:
Yes, we use this property only for tcp so i think that we should include tcp in the name